jstests/core/explain_invalid_arguments.js

```diff
@@ -0,0 +1,44 @@
+// Tests for validating arguments of the explain command.
```

Comment:
So I maybe should have been a little more specific, I apologize. I pointed you to an explain jstest example, but we prefer a different format for jstests. We prefer immediately invoked functions for jstests. See [this test](https://github.com/mongodb/mongo/blob/master/jstests/core/capped_update.js) for an example.